packages/terra-action-footer/src/terra-dev-site/doc/action-footer/Standard.a.doc.mdx

```diff
@@ -0,0 +1,59 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import SingleStartActionExample from '../example/standard/SingleStartAction?dev-site-example';
+import SingleEndActionExample from '../example/standard/SingleEndAction?dev-site-example';
+import MultipleStartEndActionsExample from '../example/standard/MultipleStartEndActions?dev-site-example';
+
+import ActionFooterPropsTable from '../../../../src/ActionFooter?dev-site-props-table';
+
+<Badge />
+
+# Terra Action Footer
+
+The `terra-action-footer` component is a footer bar that contains two sockets, start and end, for placing actionable items such as buttons and hyperlinks. If no actions are provided, the footer bar collapses to a themeable height and maintains the top border.
+
+Terra provides a Standard, Centered, and Block-style variations for Action Footer:
+* [Standard Action Footer](#usage)
+* [Centered Action Footer](/components/terra-action-footer/action-footer/centered)
+* [Block Action Footer](/components/terra-action-footer/action-footer/block/)
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-action-footer`
```

Comment:
This may be an issue with the mdx loader, but this isn't being syntax highlighed. Not a huge deal, but something we might want to look into.  

![image](https://user-images.githubusercontent.com/6720991/71629520-97959080-2bc3-11ea-942f-5fe478719bec.png)

Reply:
I played around with this, and this is the syntax style that is applied when in a list item. Moving to a paragraph / standalone renders how you'd expect